plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -62,7 +62,7 @@ public void createQueryRunner()
                 .setNodeCount(1)
                 .build();
         queryRunner.installPlugin(new HiveHadoop2Plugin());
-        temporaryDirectory = createTempDir();
+        temporaryDirectory = Files.createTempDirectory("tmp").toFile();
```

Comment:
We could change `temporaryDirectory` to be `Path` here, since nothing needs it as `File`

Reply:
`"hive.metastore.catalog.dir", temporaryDirectory.toURI().toString(),` is using toURI() for setting the config in `queryRunner.createCatalog`